src/core/agent/services/signifyNotificationService.ts

```diff
@@ -367,6 +378,32 @@ class SignifyNotificationService extends AgentService {
         await this.markNotification(notif.i);
         return;
       }
+      const notifications = await this.notificationStorage.findAllByQuery({
+        exnSaid: previousExnGrantMsg.exn.d,
+      });
+
+      if (notifications.length) {
+        const notificationRecord = notifications[0];
+        if (
+          !notificationRecord.linkedGroupRequests ||
+          !Object.values(notificationRecord.linkedGroupRequests).includes(true)
+        ) {
+          notificationRecord.linkedGroupRequests = {
+            ...notificationRecord.linkedGroupRequests,
+            [exchange.exn.d]: false,
+          };
+        } else {
+          await Agent.agent.ipexCommunications.acceptAcdcFromMultisigExn(
+            exchange.exn.d
+          );
+          notificationRecord.linkedGroupRequests = {
+            ...notificationRecord.linkedGroupRequests,
+            [exchange.exn.d]: true,
+          };
+        }
+        await this.notificationStorage.update(notificationRecord);
```

Comment:
I think we need to `markNotification` on this to make sure we've marked it as read on KERIA